crates/services/parallel-executor/src/tests/tests_executor.rs

```diff
@@ -601,3 +611,67 @@ async fn execute__utxo_ordering_kept() {
         tx2.id(&ChainId::default())
     );
 }
+
+// We use the overflow of gas to skip the transactions.
+#[tokio::test]
+async fn test_skipped_txs_fallback_mechanism() {
```

Comment:
nit: is this still an `execute__` test? technically it is `produce_without_commit_with_source`, but those are the same thing in a way.